scripts/run-docker.sh

```diff
@@ -12,8 +17,14 @@ for i in ${!args[@]}; do
       PROJ_DIR=${args[i+1]}
       unset 'args[i]'
       unset 'args[i+1]'
+    elif [[ ${args[i]} = "-xl" ]]; then
+      DOCKER_IMAGE="brainiak/rtcloudxl:latest"
+    elif [[ ${args[i]} = "-lite" ]]; then
+      DOCKER_IMAGE="brainiak/rtcloudlite:latest"
     elif [[ ${args[i]} = "-h" ]]; then
       echo "USAGE: $0 --projDir <project-dir-to-map> [list of commands to run in docker image]"
+      echo "USAGE: $0 -xl use docker image brainiak/rtcloudxl:latest"
```

Comment:
Let's list these options before the "[list of commands to run in docker image]". Then on these subsequent lines you could describe what the options do, but no need to put "USAGE: $0" again.

Reply:
does my latest push accomplish what you were thinking?